pkg/options/options.go

```diff
@@ -184,6 +185,8 @@ func (s *KubeRouterConfig) AddFlags(fs *pflag.FlagSet) {
 		"Enables Pod Networking -- Advertises and learns the routes to Pods via iBGP.")
 	fs.BoolVar(&s.RunServiceProxy, "run-service-proxy", true,
 		"Enables Service Proxy -- sets up IPVS for Kubernetes Services.")
+	fs.StringVar(&s.RuntimeEndpoint, "runtime-endpoint", "",
+		"Path to runtime socket. Used only for DSR mode")
```

Comment:
I would change the wording a bit here and add what runtimes you've specifically tested.

Maybe something like:
```
Path to CRI compatible container runtime socket (used for DSR mode). Currently known working with containerd.
```
(of course assuming that containerd is what you've tested with)